extensions/wikia/ArticleComments/classes/ArticleCommentList.class.php

```diff
@@ -1080,4 +1080,28 @@ static public function undeleteComplete($oTitle, $oUser, $reason) {
 		wfProfileOut( __METHOD__ );
 		return true;
 	}
+
+	/**
+	 * Enables commenting for an article if it is moved to a namespace that allows it
+	 * and disables it if it does not.
+	 * @param object Title $oOldTitle An object for the old article's name
+	 * @param object Title $oNewTitle An object for the new article's name
+	 * @param object User $oUser
+	 * @param integer $iOldId
+	 * @param integer $iNewId
+	 * @return bool
+	 */
+	public static function onTitleMoveComplete( Title &$oOldTitle, Title &$oNewTitle, User &$oUser, $iOldId, $iNewId ) {
+		global $wgArticleCommentsNamespaces;
+		wfProfileIn( __METHOD__ );
+
+		if ( in_array( $oNewTitle->getNamespace(), $wgArticleCommentsNamespaces ) &&
+			in_array( $oOldTitle->getNamespace(), $wgArticleCommentsNamespaces )
```

Comment:
Shouldn't this be `!in_array( $oOldTitle->getNamespace(), $wgArticleCommentsNamespaces )`? i.e. when it's moved from a namespace without comments, so that the setting is preserved if it's moved from an existing blog.